### PR TITLE
Polish English and Finnish strings

### PR DESCRIPTION
**Pull request by laurijamsa**
No major modifications. Mostly just grammar and styling mods and added one h2 heading on the Ethernet IP address page.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -183,27 +191,27 @@ <h1>
...
 					<input id="eth_dns2" type="text" placeholder="DNS 2" value="">
 				</label>
 			</div>
 
 			<div id="page-ethernet_connection-ask_user" class="hidden">
 				<div class="block-centered">
-					<span lang="en">Please connect Ethernet cable to the Ruuvi Gateway</span>
-					<span lang="fi">Yhdistä Ethernet-kaapeli Ruuvi Gateway -laitteeseen</span>
+					<span lang="en">Please connect Ethernet cable if not already connected.</span>
+					<span lang="fi">Yhdistä Ethernet-kaapeli ellei sitä ole viel kytketty.</span>
```
Comment (ojousima) [inline src/index.html:208]: vielä

File: src/index.html [hunk 2]
```diff
@@ -533,8 +545,9 @@ <h1>
 				</div>
 				<div>&nbsp;</div>
 				<div class="block-centered">
-					<span lang="en">To recover the Gateway functionality, please reboot it by pressing on the reset button.</span>
-					<span lang="fi">Käynnistä laite uudelleen laitteessa olevaa nollauspainiketta painamalla.</span>
+					<span lang="en">Please reboot the device by keeping the configuration button, located
```
Comment (ojousima) [inline src/index.html:548]: This will erase any previous configs, is this intentional?

**Review by ojousima — CHANGES_REQUESTED** [review]
Thanks, please see comments

**Review by TheSomeMan — APPROVED** [review]

**Review by ojousima — APPROVED** [review]
Thanks

**Pull request merged**